docs/maatregelen/0-org-07-intern-toezicht.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Richt een algoritmegovernance in met three lines of defence
```

Comment:
Waarom specifiek three lines of defence ? Ik ben in de praktijk ook wel four lines of defence tegen gekomen? 4e lijn is dan extern toezicht?

Reply:
Input voor de werkgroep. Voor deze release houd ik het bij wat nu uit de werkgroep en de voorbeelden naar voren is gekomen.